python/semantic_kernel/connectors/ai/mistral_ai/services/mistral_ai_config_base.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import logging
+
+from mistralai.async_client import MistralAsyncClient
+from pydantic import ConfigDict, Field, validate_call
+
+from semantic_kernel.connectors.ai.mistral_ai.services.mistral_ai_handler import MistralAIHandler
+from semantic_kernel.exceptions import ServiceInitializationError
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+class MistralAIConfigBase(MistralAIHandler):
```

Comment:
same for this as for handler, if we only use this once, then it doesn't make sense or make it more readable to separate this out.